docs/Tutorials/Basics/simple-drops.md

```diff
@@ -0,0 +1,225 @@
+
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/keypom-js-sdk/simple-example.js#L8-L25
+```
+
+---
+
+## Creating the Simple Drop
+Now that the connection to the NEAR blockchain is set up, the next step is to create the Keypairs and then create the drop.  
+:::tip
+Recall that the Keypairs that are being generated are the ones that will be embedded in the link given to the user. 
+:::
+
+### The Keypom SDK Approach
+In the SDK, this process starts with calling the `initKeypom` function. This function will always be the first function you need to call to interact with the SDK. 
+
+:::info
+The `initKeypom` function is the first function you need to call in order to interact with the SDK.
+:::
+
+After `initKeypom` is called, the key generation and drop creation can be done. These two tasks are combined into one function, `createDrop`. This function can be either given a set of pre-created keypairs, **or** it can create them automatically.
+
+### The NEAR-API-JS Approach
+
+With NEAR-API-JS, these two tasks are seperate, as the `create_drop` function only creates the drop. The keys that are passed into it must be generated seperately.These two tasks are separate and pre-created keys *must* be passed into create_drop
+
+:::tip
+Note that with NEAR-API-JS, an attached deposit must be added to the `functionCall`. This is to cover the cost of creating the drop. With simple drops, the cost is the sum of $NEAR embedded in each link as well as the storage costs on the protocol.
+
+In these tutorials, a generous deposit is attached and any unused $NEAR is simply added to the funder's [Keypom balance](../../Concepts/Keypom%20Protocol/balances.md)
+:::
+
+<Tabs>
+<TabItem value="KPJS" label="🔑Keypom-JS SDK">
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/keypom-js-sdk/simple-example.js#L27-L41
+```
+
+</TabItem>
+<TabItem value="NRJS" label="💻NEAR-API-JS">
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/near-api-js/simple-near-example.js#L26-L51
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+All function parameters and default values for the SDK and Keypom functions can be found in the [SDK Typedocs](../../keypom-sdk/modules.md). For NEAR-API-JS functions, their arguments can be found [here](https://docs.near.org/tools/near-api-js/reference)
+:::
+
+---
+
+## Full Solution
+The final code, using the skeleton defined in the [introduction](simple-drops.md#introduction) to create a simple 1 $NEAR linkdrop can be seen below.
+
+<Tabs>
+<TabItem value="KPJS" label="🔑Keypom-JS SDK">
+
+```js reference
```

Comment:
We need to add a section about generating the linkdrop links cuz this shows up in the final code but isn't mentioned anywhere in the tutorial. We should talk about how for this tutorial, we'll use the NEAR wallet which has a dedicated claim page that expects a URL of the form:

```
wallet.NETWORK.near.org/linkdrop/CONTRACT/PRIVATE_KEY
```